source/tests/system/nifgen_driver_api_tests.cpp

```diff
@@ -366,20 +366,6 @@ TEST_F(nifgenDriverApiTest, ConfigureTriggerMode_ConfiguresSuccessfully)
   EXPECT_EQ(expected_value, actual_trigger_mode);
 }
 
-TEST_F(nifgenDriverApiTest, SendSoftwareTrigger_TriggersSuccessfully)
```

Comment:
Is there any other system test from the same category as `SendSoftwareTrigger`? I want to make sure that we are not loosing test coverage by removing this obsolete API.

Reply:
There is another API called `'SendSoftwareEdgeTrigger'`, metadata for which is yet to be added, maybe we can reuse this test for that API. The obsolete header says we should use that function instead of `SendSoftwareTrigger`: 

![image](https://user-images.githubusercontent.com/47028785/120785025-59379a00-c54a-11eb-9cf6-feb6b92e2369.png)

Reply:
The alternative of that API, `SendSoftwareEdgeTrigger` is not yet added. I created a task to track this. Once the api gets added, I will add back this test using that API